openshift/templates/applications/routes/request-api.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: request-api-template
+message: |-
+  A Jenkins server must be instantiated in this project to manage
+  the Pipeline BuildConfig created by this template. You will be able to log in to
+  it using your OpenShift user credentials.
+metadata:
+  annotations:
+    openshift.io/display-name: request-api template
+    description: |-
+      This is a template for request-api.
+    iconClass: icon-jenkins
+    tags: instant-app,jenkins
+  name: request-api-template
+parameters:
+- description: The OpenShift Namespace prefix.
+  displayName: Namespace Prefix (Prefix for each namespace ie prefix-tools, prefix-dev etc)
+  name: namespacePrefix
```

Comment:
Should probably have consistent cases of parameters here, either all camel case or all upper